scripts/Decay_baseline_correct.R

```diff
@@ -0,0 +1,123 @@
+#script to extract values for each pulse and copy to csv file: max > 40 seconds - for BASELINE CORRECTED habit delF values (supplied from habit_baseline_pulses function)
+#also determines t1/2 for each calcium trace
+#open wt_habit_baseline_pulses .csv file
+#on plots - dashed lines = fit
+decay_baseline_correct <- function(genotype, #mandatory genotype, should be text
```

Comment:
See comment above, you should make the filename = the function name. In this case 'D' is capitalized.